### PR TITLE
Linktocentral

### PR DESCRIPTION
**Pull request by Kawohl**
added a link to central owncloud.org in owncloudtheme.cpp for the settings overview

### CODE REVIEW

File: src/libsync/owncloudtheme.cpp [hunk 1]
```diff
@@ -45,8 +45,9 @@ QString ownCloudTheme::about() const
 {
     QString devString;
     devString = trUtf8("<p>Version %2. For more information visit <a href=\"%3\">%4</a></p>"
-               "<p><small>By Klaas Freitag, Daniel Molkentin, Jan-Christoph Borchardt, "
-               "Olivier Goffart, Markus Götz and others.</small></p>"
+               "<p>For known issues and help, please visit: <a href=https://central.owncloud.org/c/help/desktop-file-sync>central.owncloud.org</a></p>"
+               "<p><small>By Klaas Freitag, Daniel Molkentin, Olivier Goffart, Markus Götz, "
```
Comment (dragotin) [inline src/libsync/owncloudtheme.cpp]: Might be just me being old-school, but I more prefer reading urls including the protocol in user visible strings, that would mean `central.o.o` => `https://central.o.o`.... just if you want.

Comment (Kawohl) [inline src/libsync/owncloudtheme.cpp]: fine with me!

Comment (guruz) [inline src/libsync/owncloudtheme.cpp]: The "" are missing around the https:// .. :)

**Comment by guruz** [comment]
(To whoever merging this after @Kawohl fixes the " issue: Please use "Squash and merge" for clean history)

**Comment by Kawohl** [comment]
imho the : ""  are unnecessary and confusing... The Links work and are clickable
please explain what can be done better and why.

Best John

**Comment by guruz** [comment]
@Kawohl but it's not valid XML/HTML.. I can fix it myself and merge https://github.com/owncloud/client/pull/5271/files#diff-5c233b67dd98ce7e8ea3ffbe0a4b3547R48

**Comment by felixboehm** [comment]
@Kawohl welcome to the show!

File: src/libsync/owncloudtheme.cpp (hunk 1 above)
Comment (Kawohl) [inline src/libsync/owncloudtheme.cpp]: @guruz i get the line but i just can't figure out the right way... could you please point me somewhere for an example?
Thanks!

**Pull request closed without merging**